src/metabase/util.cljc

```diff
@@ -247,7 +247,12 @@
          (subs s 1))))
 
 (defn snake-keys
-  "Convert the keys in a map from `kebab-case` to `snake_case`."
+  "Convert the top-level keys in a map to `snake_case`."
+  [m]
+  (update-keys m ->snake_case_en))
+
+(defn deep-snake-keys
+  "Convert the keys in a map to `snake_case`."
```

Comment:
nit - probably obvious from the function name but `keys in a map` alone to me implies top-level 🤷‍♂️ 
```suggestion
  "Recursively convert the keys in a map to `snake_case`."
```

Reply:
Ah, good call. I copied the old docstring, but I like yours better.

Reply:
fixed